### ISSUE TITLE
Crates spawning on slopes are cut off

### ISSUE THREAD

**Issue by Mailaender** [OpenRA/OpenRA#18017]
![image](https://user-images.githubusercontent.com/756669/80861994-14b83b80-8c72-11ea-89a7-7eafcac257f6.png)

**Comment by pchote** [comment]
I can't reproduce this, so possibly a duplicate of #18015?

**Comment by Mailaender** [comment]
I thought maybe we don't want them to spawn on slopes at all.

**Comment by pchote** [comment]
The wording suggests that this happens for all slopes - can you confirm whether this is indeed the case, or was this something that you only saw once? Some maps have tile errors that cause discontinuities in the map - if a crate is sitting behind one of these breaks it will (correctly!) be partly hidden.

An example of this can be seen in the bottom-right of the "Oasis Trouble" map:
<img width="772" alt="Screenshot 2020-05-03 at 13 01 39" src="https://user-images.githubusercontent.com/167819/80913688-6e863780-8d3e-11ea-92e3-8129df27bc41.png">

**Comment by Mailaender** [comment]
I saw this only once and the slope might be incorrect. However I walked infantry on it and it seemed not obviously wrong.
